### ISSUE TITLE
Accessibility issue for web pages

### ISSUE THREAD

**Issue by wasimdevapps** [primefaces/primeng#11650]
### Describe the bug

We need to fix issue like “Button must have discernible text” as this violates accessibility. So we have found in Ng prime grid paginator button doesn’t have tags like (aria-label etc) where the issues will resolve.

### Environment

Development

### Angular version

8.0.0

### PrimeNG version

13.4.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

16.15.1

### Browser(s)

chrome

### Expected behavior

accessibility issue should not appear while running axe tool.

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,
